ReleaseProcessAutomation.UnitTests/ReleaseProcessScript.Test.Config

```diff
@@ -24,7 +24,10 @@
         <fileName>Build/Customizations/Version.props</fileName>
     </developStableMergeIgnoreList>
     <msBuildSettings>
+        <!--MsBuildPathTestPlaceholder-->
         <msBuildPath>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Current\Bin\MSBuild.exe</msBuildPath>
+        <alternativeMSBuildPath>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Current\Bin\MSBuild.exe</alternativeMSBuildPath>
+        <alternativeMSBuildPath>C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\MSBuild.exe</alternativeMSBuildPath>
```

Comment:
We do we have the same entry twice? Wouldn't it be enough to have one entry for the Professional edition?